core/internal/cltest/cltest.go

```diff
@@ -942,6 +935,21 @@ func WaitForRuns(t testing.TB, j models.JobSpec, store *strpkg.Store, want int)
 	return jrs
 }
 
+// AssertRunsStays asserts that the number of job runs for a particular job remains at the provided values
```

Comment:
This seems to have already been merged into develop: https://github.com/smartcontractkit/chainlink/blob/develop/core/internal/cltest/cltest.go#L938

Seems like merging will possibly add a duplicate copy?